generator/templates/Authorisation/src/views/Login.vue

```diff
@@ -9,16 +9,15 @@
     </v-row>
 </template>
 
-<script>
+<script lang="js">
```

Comment:
Waarvoor is dit nodig?

Reply:
Not necessary, but this will tell the IDE that everything should be interpreted as javascript instead of typescript